CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
-### 2.0.0 (Next)
+### 1.0.1 (Next)
 
-* [#62](https://github.com/dblock/strava-ruby-client/pull/68): Drops `Activity#type` attribute as it is being deprecated by Strava, dropping `Activity#type_emoji` with it - [@simonneutert](https://github.com/simonneutert).
```

Comment:
If you want to make a 1.0.1 fix without making this breaking change, make a 1.x branch from an older commit in this repo, then PR changes into it. Either way make a PR into `master` first, so here you want to put back these lines.

Reply:
thank you, I was off a commit! Hope it's correct now.